examples/record-match-statement/record_match_statement.bal

```diff
@@ -52,8 +52,8 @@ function basicMatch(any a) {
     }
 }
 
-// Following method uses structured record match patterns with different fields along with a type guards. The given
-// `match` expression will be checked for "isLike" relationship and also it will check the type guard present for a
+// Following method uses structured record match patterns with different fields along with type guards. The given
+// `match` expression will be checked for "isLike" relationship and it will also check the type guard present for a
```

Comment:
We are checking a value to see if it matches any of the patterns. Not checking the `match` expression itself. We are checking if the value "is-like" one of the provided patterns. The use of type guards provide more fine grained control over the constraints of the "is-like" match. @MaryamZi thoughts?

Reply:
How about something like

"The expression used with `match` will be evaluated to check if it is like any of the `match` patterns. For a successful match an additional check will also be performed against the specified type guard."

Reply:
This looks good to me.. However, feel that is like should be emphasized.. maybe like: "is-like"

Reply:
Addressed with 259c088